### DISCUSSION TITLE
Bricks sortable?

### DISCUSSION THREAD

**Discussion by niederberger-qw** [pimcore/pimcore#13210] [Q&A / Support]
Hi there,

are the Objectbricks sortable in the data view?

![grafik](https://user-images.githubusercontent.com/85213755/191294006-f1b58b3a-7b84-4413-b8e6-8a6d540d12d9.png)
I think it would be great, if there was a sorter integer field or something in the objectbricks general settings.

Maybe I missed some function here?

**Comment by kingjia90** [comment]
Hi,

At the moment, it's not sortable.
There is a similar issue https://github.com/pimcore/pimcore/issues/4388 suggesting it as a new feature.
